pkg/nvlib/info/info.go

```diff
@@ -27,15 +27,22 @@ import (
 )
 
 type infolib struct {
+	logger basicLogger
+	Properties
+	Resolver
+}
```

Comment:
I chose `Properties` because the interface implements a set of `bool, string` functions that return information about the system. These system properties are then used to make decissions -- such as determining the platform.

I'm happy to discuss alternative names.

Reply:
The top-level logger is not needed and has been removed. I have also renamed `Properties` to `ProperyExtractor` in the lates revision.